### ISSUE TITLE
Assertion Failed - "...Mission names specified..."

### ISSUE THREAD

**Issue by bisto67** [eezstreet/SWATEliteForce#325]
Hello, I am getting the error 

`The number of campaign MissionNames specified in SwatGUIState.ini does not match the number of FriendlyNames`

If I select "OK" I then get this:

`[tcohen] ProcedureClass index 4 in the Procedures configuration file is not a valid Procedure class.`

Selecting "OK" again, then gives me this:

```
No Label

Crash Time: 10/15/17 18:21:45

OS: Windows XP 5.1 (Build: 2600)
CPU: AuthenticAMD Unknown processor @ 3507 MHz with 1023MB RAM
Video: NVIDIA GeForce GTX 750 Ti (8205)
 
UserName: danny
ComputerName: DANNYSPC
Application location: C:\GOG Games\SWAT 4\ContentExpansion\System\

SWAT Build Number: 6.1

Procedures Transient.Procedures0 (Function SwatGame.Procedures.Construct_Void:00F3) Assertion failed, line 26

History: FFrame::Serialize <- UObject::CallFunction [SCRIPT='SwatGame.Procedures.Construct_Void' OBJ='Procedures0'] <- UObject::ProcessEvent [Function SwatGame.SwatRepo.PostBeginPlay() on SwatRepo Transient.SwatRepo0] <- BeginPlay <- UGameEngine::LoadMap <- UGameEngine::Init <- InitEngine
```

It may be worth noting I am using Windows 10. I have selected compatibility with Windows XP SP2 and given Admin privileges.

**Comment by eezstreet** [comment]
This bug is fixed in v6.3.

**Comment by dylmye** [comment]
Hi, getting the same issue with v7.0.

```
No Label

Crash Time: 12/30/20 23:22:53

OS: Windows NT 6.2 (Build: 9200)
CPU: AuthenticAMD Unknown processor @ 3200 MHz with 2047MB RAM
Video: NVIDIA GeForce GTX 1060 6GB (6089)

UserName: dylan
ComputerName: [...]
Application location: [...]\SWAT 4\ContentExpansion\System\

SWAT Build Number: 7.0

Procedures Transient.Procedures0 (Function SwatGame.Procedures.Construct_Void:00F3) Assertion failed, line 26

History: FFrame::Serialize <- UObject::CallFunction [SCRIPT='SwatGame.Procedures.Construct_Void' OBJ='Procedures0'] <- UObject::ProcessEvent [Function SwatGame.SwatRepo.PostBeginPlay() on SwatRepo Transient.SwatRepo0] <- BeginPlay <- UGameEngine::LoadMap <- UGameEngine::Init <- InitEngine
```
